docs/specifications/tests/Nameserver-TP/nameserver14.md

```diff
@@ -0,0 +1,77 @@
+# NAMESERVER14: Test for unknown version with unknown OPTION-CODE
+
+## Test case identifier
+
+**NAMESERVER14** 
+
+### Objective
+
+This test case actually combines the test options in test cases
+[NAMESERVER10](../nameserver10.md) and [NAMESERVER11](../nameserver11.md). 
```

Comment:
What does this Test Case add? Isn't everything covered by the other two test cases?

Reply:
I did have that question. But in the EDNS compliance tests, they are doing a separate test by combining both the options (unknown option code and version). 

If @vlevigneron also thinks that this test is not needed, we can remove it.

Reply:
@sandoche2k, does the test case specification really catch that?

Reply:
@matsduf Updated. Pls. verify.